service/service.go

```diff
@@ -169,6 +169,7 @@ func New(config Config) (*Service, error) {
 			OIDC:             OIDC,
 			InstallationName: config.Viper.GetString(config.Flag.Service.Installation.Name),
 			ProjectName:      config.ProjectName,
+			RegistryDomain:   config.Viper.GetString(config.Flag.Service.RegistryDomain),
```

Comment:
Would it be possible to have default values to avoid panicking later? I believe we will use `quay.io` in most cases.

Reply:
I think flag registration in `main.go` should provide the default, no?

Reply:
Yes, I missed that entirely. Never mind!